bolt/runtime/CMakeLists.txt

```
@@ -35,7 +35,12 @@ set(BOLT_RT_FLAGS
   -fno-exceptions
   -fno-rtti
   -fno-stack-protector
-  -fPIC)
+  -fPIC
+  # Runtime currently assumes omitted frame pointers for functions marked __attribute((naked)).
+  # Protect against distros adding -fno-omit-frame-pointer and compiling with GCC.
+  # Refs: llvm/llvm-project#147569 https://gcc.gnu.org/bugzilla/show_bug.cgi?id=77882
```

TMU, #147569 is unrelated so it could be dropped? ([link](https://github.com/llvm/llvm-project/issues/147569#issuecomment-3063346038))

```suggestion
  # Refs: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=77882
```

---

Thanks, I created https://github.com/llvm/llvm-project/issues/148595 and I'll hide comments relating to this problem on #147569, but I will leave it as 'see also' in the commit header.